manifests/init.pp

```diff
@@ -29,7 +29,10 @@
 #   generated config.
 #
 # * `download_url`
-#   URL to download the vault zip distribution from.
+#   Manual URL to download the vault zip distribution from.
+#
+# * `download_url_base`
```

Comment:
Can you please also update the `README.md` to reflect the new params?

I've looked into using `puppet-strings` for this but it doesn't quite support generating markdown out of the box so I'm just manually keeping things in sync for now.